wolfssl/wolfcrypt/asn_public.h

```diff
@@ -879,6 +879,9 @@ WOLFSSL_API int  wc_ParseCert(
 
 WOLFSSL_API int wc_GetPubKeyDerFromCert(struct DecodedCert* cert,
                                         byte* derKey, word32* derKeySz);
+WOLFSSL_API int wc_GetSubjectPubKeyInfoDerFromCert(const byte* cert,
```

Comment:
`cert` -> `certDer` and `pubKey` -> `pubKeyDer` to match function implementation.